translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionCharSequenceAssertion.kt

```diff
@@ -19,6 +19,7 @@ enum class DescriptionCharSequenceAssertion(override val value: String) : String
     IGNORING_CASE("%s, ignoring case"),
     MATCHES("matches entirely"),
     MISMATCHES("does not match entirely"),
+    NOT_FOUND("but no such item was found."),
```

Comment:
Or potentially: `"but no matches were found"`

Reply:
Good idea, I think for CharSequence `but no match was found` makes more sense, then the report looks like the following:
```
expected that subject: "Hello People"        <1243155369>
◆ contains: 
  ⚬ value: "World"
     » but no match was found
```

Reply:
Please run `./gradlew :readme-examples:build` after the change (or `./gradlew build` which includes it), I think there should be changes in the README due to your change

Reply:
Will do. Can I get a translation for the `but no match was found` description as well? :smile:

Reply:
Sure, `aber es wurde kein Treffer gefunden`